.gitignore

```diff
@@ -47,3 +48,5 @@ src/core/conf/app.conf
 
 src/server/v2.0/models/
 src/server/v2.0/restapi/
+pkg/
+pkh/**
```

Comment:
Could you remove these two lines because these two directories not be generated during the development?

Reply:
Done